server/lib/nl/config_builder/timeline.py

```diff
@@ -108,7 +107,9 @@ def single_place_single_var_timeline_block(column, place: Place, sv_dcid: str,
                                            sv2thing: types.SV2Thing,
                                            single_date: types.Date,
                                            date_range: types.Date,
-                                           cv: ChartVars):
+                                           sv_place_facet_id: Dict[str,
+                                                                   Dict[str,
+                                                                        str]]):
```

Comment:
Should we use a dataclass for this since it gets passed around a lot?

Reply:
I'm not sure how this dataclass will look because there aren't really fields we're accessing, we just want the map of sv -> place -> facetid .. 

One thought is that I could pass in ChartSpec instead and access chartspec.sv_place_facet_id to make things more clear? wdyt?

Reply:
I see.  In C++ to make this case more readable we'd just use typedef. So should we try:

`Place2Facet = Dict[str, str]`
`Sv2Place2Facet = Dict[str, Place2Facet]`

So just replace the type annotations with `Sv2Place2Facet`, impl doesn't change.

Reply:
oh, super cool, didn't know that worked in python, done!